sessions/css-grid/demo-end/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>CSS Grid Demo - Bootcamps</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <main class="bootcamp-grid">
+      <section class="bootcamp bootcamp--1">
+        <h2 class="bootcamp__headline">Web Development</h2>
```

Comment:
I think, it would be nice to add an `h1` element as there is none on this page. 🙂